README.md

```diff
@@ -72,7 +72,8 @@ The only required field for the metadata file is the srcVideo and the workflow w
     "MediaConvert_Template_2160p":"string",
     "MediaConvert_Template_1080p":"string",
     "MediaConvert_Template_720p":"string",
-    "JobTemplate":"custom-job-template"
+    "JobTemplate":"custom-job-template",
+    "InputRotate": "DEGREE_0"
```

Comment:
https://docs.aws.amazon.com/mediaconvert/latest/apireference/jobs.html#jobs-model-inputrotate 

Should also probably mention using anything but default is going to cause the profiler to possibly select the wrong encoding profile as it's pixel math will be utilizing the pre-rotated dimensions. 

For example, all ios videos record portrait (without the developer normalizing) videos with `ROTATE 90`,  if `AUTO` were to be selected, without overriding the `JobTemplate`, the profiler doesn't know what the final dimensions after rotation may be. 

If you know you have 16:9 portrait videos with ROTATION 90, you will need to intervene to help profiler out 

```javascript
if (event.srcWidth > event.srcHeight) {
      console.log('This video is most likely going to get rotated, swap width/height')
      event.srcWidth = mediaInfo.video[0].height;
      event.srcHeight = mediaInfo.video[0].width;
 }
```